apparmor.d/abstractions/vulkan.d/complete

```diff
@@ -1,6 +1,4 @@
 # apparmor.d - Full set of apparmor profiles
 # SPDX-License-Identifier: GPL-2.0-only
 
-  /etc/glvnd/egl_vendor.d/{,*.json} r,
```

Comment:
Do not remove this yet. It has already been [upstreamed](https://gitlab.com/apparmor/apparmor/-/merge_requests/995) yes. But some distribution (Debian) may not have been updated yet.